.github/ISSUE_TEMPLATE/config.yml.txt

```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Documentation Issues
+    about: Report issues with the documentation for O3DE
```

Comment:
Small detail: from the perspective of ROS 2 Gem: most documentation is indeed (and will be) hosted from o3de.org. Some other documentation (design docs) are not likely to be subject to bugs. Would doxygen documentation also fall under this category? If not, perhaps no adaptation is needed here.

Reply:
```suggestion
    about: Report issues with the O3DE.org/docs documentation
```